### ISSUE TITLE
﻿Change opacity levels

### ISSUE THREAD

**Issue by 2dvision** [opentoonz/opentoonz#490]
If you want inking, you need change opacity "sketch level" for inking in "ink level" be easy. 

now you can use FX, but this work only on preview mode viewer.

**Comment by blurymind** [comment]
It is already possible to do that, but it is somewhat hidden.
https://www.reddit.com/r/OpenToonz/comments/4d4kgs/any_idea_on_how_to_change_a_levels_overall_opacity/

**Comment by shun-iwasawa** [comment]
In the comment in reddit says:
> In the top of the column you have a yellow button, if you click it the col are made transparent and if you click and hold an transparency bar appears. 

To be precise, you need to click&hold the "orange" button (=camera stand visibility toggle) in order to make levels semi-transparent in the viewer. This feature is available in the current version as well.

![opacity_slider](https://cloud.githubusercontent.com/assets/17974955/16264848/d88dfb94-38b6-11e6-8527-ab83710d3a54.png)

**Comment by 2dvision** [comment]
OH GOD! :)))

Whay its hidden? OT is not a video game. Users should be able to easily see it. Perhaps with an icon, percent or etc. What do you think?

**Comment by blurymind** [comment]
the problem is that buttons in OT tend to be sometimes invincible when in a disabled state.

This is similar to this: https://github.com/opentoonz/opentoonz/issues/411
Buttons for level visibility also disappear when disabled. They should have a disabled state instead of disappearing.

In this case it is a slider though

**Comment by blurymind** [comment]
in defense of the current design - it does make for less clutter in the x-sheet. That makes it  extremely clean and sleek if you compare it to tvpaint.

**Comment by AyameLee** [comment]
What about an icon? Does it look too busy?

![col_tz](https://cloud.githubusercontent.com/assets/18118611/16291310/615ed484-38fd-11e6-9b34-21c89695c19d.png)

**Comment by blurymind** [comment]
@AyameLee @2dvision  technically the entire 'camera stand visibility toggle' button IS that 'opacity' button! The gotcha for the user is the fact that the button needs to be clicked and held in order for the opacity slider to appear.
So adding another button next to it is kind of redundant in this case and decreases the clicking area.

Right now we have a button that does two things - depending on whether its clicked or clicked and held.

What you are asking for is to split the button into two buttons that are simply clicked and each serves  one function.

We could hint to the user that the toggle visibility button is also a slider, by indicating percentage of opacity:
![sliderhint](https://cloud.githubusercontent.com/assets/6495061/16303690/ef2e2872-3949-11e6-90e5-91081ced0b45.png)

**Comment by AyameLee** [comment]
I got the opinion of some UX designers and opened an issue on their feedback under #513. Generally there is an agreement that there should be some kind of "obvious user facing invocation". They're not animators, but it was interesting to see their feedback, stuff like high contrast icons isn't something I would have thought of.

**Comment by blurymind** [comment]
The button holding design is good for tablet users, but it is somewhat hard to discover as not a lot of other software has it

**Comment by shun-iwasawa** [comment]
When I customized Toonz Ghibli edition, I decided to follow the UI in the existing software (Toonz v4.6) in order to keep consistency. In Toonz v4.6 it also had large yellow "render visibility toggle" button on top of the column head like this:

![46interface](https://cloud.githubusercontent.com/assets/17974955/16326418/6b1f5304-3a01-11e6-8c76-a6cbb8c9fd55.png)

I and other Ghibli stuffs loved this colored-and-large indicator since you can see which columns are activated in rendering at a glance. ( I would like to think it was not only because of aging workforce in the studio. )

**Comment by Rayek** [comment]
I tend to agree with Shun-Iwasawa: those tiny icons will be much harder to differentiate than the current version. It is obvious which columns are active, and which not for both table and render visibility. That would not be the case when tiny icons would be implemented. 

Secondly, has anyone ever worked with a tablet/cintiq? The current hit areas are four-five times larger than the tiny icons as suggested in #513 . I would hate working with such tiny buttons.

In short, the current GUI might not look "nice", but is quite utilitarian and functional. And it is easy to distinguish which columns are active and which are not with a mere glance. 

Aside from the 'looks', tiny icons would be a step backwards, in my opinion.
